src/shared/constants/index.ts

```diff
@@ -132,3 +132,6 @@ export const LEGEND_ORIENTATION_THRESHOLD_DEFAULT = 10
 
 export const QUERY_BUILDER_MODE = 'builder'
 export const SCRIPT_EDITOR_MODE = 'advanced'
+
+export const MARKDOWN_UNSUPPORTED_IMAGE =
```

Comment:
it seems like this is obvious from context but just so it's clear: the business rule is we don't support images in markdown on cloud, but they're totally okay in oss environments. if we start integrating ui into OSS, this will probably become relevant again.

Reply:
exactly! Let me add that to the PR description to make sure it's clear